variable/technology/technologies.yaml

```diff
@@ -186,6 +189,74 @@ Capital Cost|Liquids|Oil:
    description: Capital cost of a new oil refining plant
    unit: US$2010/kW
 
+Constant Variable Cost|Electricity|Coal:
```

Comment:
The name "constant variable ..." is a self-contradiction, how about `Variable Cost (Intercept)|...`?